src/test/java/org/skife/jdbi/v2/sqlobject/TestGetGeneratedKeysPostgres.java

```diff
@@ -67,9 +72,10 @@ public Object withHandle(Handle handle) throws Exception
         public String findNameById(@Bind long id);
     }
 
-    @Ignore
     @Test
     public void testFoo() throws Exception {
+        assumeTrue(postgresIsInstalled);
```

Comment:
I tested again and it looks like assumptions actually work in `@Before`. The problem is that the test is failed if I place an assumption is the `@After` method, because JUnit converts `AssumptionViolatedException` to `MultipleFailureException`. So we can't use `Assume.assumeTrue` twice in the same test.